### DISCUSSION TITLE
make package

### DISCUSSION THREAD

**Discussion by yuioto** [xfangfang/wiliwili#72] [Q&A]
# 让cmake准备将wiliwili安装进系统。

## 生成desktop文件在build文件夹
REMADE中提到过可以生成desktop文件在桌面。
能允许生成desktop文件在build文件夹以为包制作做准备？

## Cannot load file.
只在/usr/local/share 中寻找运行必要的文件
我想要通过修改编译变量让wiliwili在/usr/share中寻找文件。

**Comment by xfangfang** [comment]
应该可以使用 -DCMAKE_INSTALL_PREFIX=/usr/local 来指定安装目录

---

如果您的母语不是中文，可以使用英语和我交流的，有些字句我不是很明白您说的是什么意思。: )

**Reply by yuioto** [reply]
谢谢。我只会中文来着😂,外语还挺不好的。想说正式点的语气(⁠ ⁠；⁠∀⁠；⁠)

您的理解是对的，待会我编辑一下，改正常点。

**Comment by xfangfang** [comment]
这是我用到的 desktop 文件，你可以直接使用或者修改他: https://github.com/xfangfang/wiliwili/blob/yoga/scripts/linux/cn.xfangfang.wiliwili.desktop
